android/app/src/main/res/layout/item_bookmark_category.xml

```diff
@@ -48,12 +50,14 @@
   <ImageView
     android:id="@+id/more"
     android:layout_width="wrap_content"
-    android:layout_height="match_parent"
+    android:layout_height="wrap_content"
     android:layout_alignParentEnd="true"
     android:layout_centerVertical="true"
     android:background="?selectableItemBackgroundBorderless"
     android:importantForAccessibility="no"
     android:paddingHorizontal="@dimen/margin_half"
+    android:paddingTop="22dp"
+    android:paddingBottom="24dp"
```

Comment:
What are the magic values?
Why not to have the same padding for all dimensions? `android:padding="..."`

Reply:
Actually i calculated the top and bottom paddings from the text's padding to fit the icons exactly in center.

Reply:
I don't think it should be solved this way. How about other screen resolutions - bigger/smaller screens, tablets? Have you checked it in the landscape mode?
I don't really like this approach but if it looks OK for different screens then OK :)

Reply:
I will test it on different screens.
But technically increasing padding is just increasing the button area so that click can be done easily. For different screens i guess it will adjust itself accordingly, but still i will look for some other better solution with lesser code changes.